assets/terraform-modules/aws/flatcar-linux/kubernetes/ami.tf

```diff
@@ -1,14 +1,14 @@
 locals {
   # Pick a Container Linux derivative
-  ami_id = "${element(concat(data.aws_ami.flatcar.*.image_id, data.aws_ami.coreos.*.image_id, list("")), 0)}"
```

Comment:
Hm, rather than fixing a syntax here, I'd remove CoreOS support, this will allow us to use `image_id` directly. Which will also partially address #324.

Reply:
Good point. Happy to do so. Same as #1246.